capstone/src/main/webapp/script.js

```diff
@@ -287,44 +290,55 @@ async function getInputFilters() {
 
   // do manual search if filter is selected, else do Nearby Search w Places API
   if (_showSmallBusiness || _showBlackOwnedBusiness) {
-    if (!isStringEmpty(keyword)) {
-      const keywordEntities = getEntities(keyword);
-      // TODO(#47): manual search
-    }
+    await(manualSearch(keyword));
   } else {
     await getPlacesSearchResults(keyword);
   }
+  _showSmallBusiness = false;
+  _showBlackOwnedBusiness = false;
   document.getElementById('search-button').disabled = false;
 }
 
 /** @param {String} str input string to check
-    @return {boolean} if str is empty, contains only white space, or null */
+    @return {boolean} if str is empty, contains only white space, or null 
+ */
 function isStringEmpty(str) {
   return (str.length === 0 || !str.trim() || !str);
 }
 
-async function getReviewsEntities(reviews) {
-  // TODO(#33): integrate with actual reviews of businesses
-  const reviewsEntities = await getEntities(reviews);
-  return reviewsEntities;
+/** @returns{Object} gets top 20 results, sorted by avg review,
+    matching @param {String} keyword 
+ */
+async function manualSearch(keyword) {
+  if (_showSmallBusiness) {
+    console.log("doing manual search");
+    return fetch('/small-restaurants?keyword='+keyword).then((response) => response.json())
+      .then((restaurantResults) => {
+        restaurantResults.forEach((place) => {
+          setMarkerManualSearch(place);
+        })
+      });
+  }
+  else if (_showBlackOwnedBusiness) {
```

Comment:
nit: you don't need the `else` since we're returning in the other if block so it won't get here if `_showSmallBusiness` is true